### PR TITLE
SHO-77: Footer

### PR DESCRIPTION
**Pull request by saulprl**
## Describe your changes

This PR adds a new component: Footer. Additionally, it fixes some of the reported issues/nice-to-haves.

## Task URL (Asana, Jira, etc...)

- [SHO-77](https://cultoperrocafe.nifty.pm/l/xpZo29w6lzaS)

#### Additional tasks

- [SHO-63](https://cultoperrocafe.nifty.pm/l/6phvwv6wkSak)
- [SHO-64](https://cultoperrocafe.nifty.pm/l/GTXokWCwrfoC)
- [SHO-76](https://cultoperrocafe.nifty.pm/l/qg7yMtMtRyucM)
- [SHO-80](https://cultoperrocafe.nifty.pm/l/6qHtw7VwMsd!G)

## What was done in this pull request

- [x] Added new component: `<Footer />`.
- [x] Updated `Cult` component so its background takes 100% of the available width.
- [x] Updated the Mobile NavBar to make links easier to tap.
- [x] Modified the GraphQL query on `index.tsx` so the retrieved products' images are higher quality.
- [x] Storybook with its only variant.
- [x] Added component to `index.tsx`.

## Demo / Screenshot / Video

[SHO-77.webm](https://user-images.githubusercontent.com/39208827/234462781-b9554815-76ae-4d24-9ec1-b8b2dabf1c11.webm)

SHO-64

[SHO-63.webm](https://user-images.githubusercontent.com/39208827/234464097-c887d9d5-7a03-46e4-a8b6-3ed765e1e022.webm)

### CODE REVIEW

File: app/components/templates/footer/footer.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+export const Subtitle = styled(Paragraph)`
+  color: ${({theme}) => theme.colors.backgroundDarker};
+  font-family: ${({theme}) => theme.fonts.title};
+  text-transform: uppercase;
+`;
+
+export const LightParagraph = styled(Paragraph)`
+  color: ${({theme}) => theme.colors.backgroundDarker};
+  font-size: 12px;
```
Comment (arielplas) [inline app/components/templates/footer/footer.styles.tsx]: IMO -> this is too small. But maybe I need new glasses....


what do you think? @tonymtz @jmsalcido

File: app/components/templates/footer/footer.component.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            <Subtitle>Las Redes</Subtitle>
+            <SocialMediaWrapper>
+              <InstagramButton href={configData.socialMedia.instagram} />
+              <FacebookButton href={configData.socialMedia.facebook} />
+              <TwitterButton href={configData.socialMedia.twitter} />
+            </SocialMediaWrapper>
+          </Container>
+        </TopContainer>
+        <BottomContainer>
+          <Img src={sparkle} alt="Sparkle graphic" />
```
Comment (arielplas) [inline app/components/templates/footer/footer.component.tsx]: Our website is in spanish, change the alt to be in spanish 😄

File: app/components/templates/footer/footer.component.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+          <CopyrightParagraph>{configData.copyright}</CopyrightParagraph>
+          <PositionedLogo src={logo} alt="Culto al Perro Café logo" />
```
Comment (arielplas) [inline app/components/templates/footer/footer.component.tsx]: Same here

File: app/components/templates/footer/footer.styles.tsx (hunk 1 above)
Comment (saulprl) [reply app/components/templates/footer/footer.styles.tsx]: I agree. Honestly I just took the values from Figma, but I think it'd look better using the same font-size values as the NavBar, for instance

File: app/components/templates/footer/footer.component.tsx (hunk 2 above)
Comment (saulprl) [reply app/components/templates/footer/footer.component.tsx]: I've updated all of the hardcoded alt props in the project and set the HTML lang to `es`. I'll wait for review on the font size issue before pushing those changes, though.

**Review by tonymtz — APPROVED** [review]

**Pull request merged**